source/extensions/filters/network/redis_proxy/conn_pool_impl.cc

```diff
@@ -56,7 +56,7 @@ InstanceImpl::ThreadLocalPool::ThreadLocalPool(InstanceImpl& parent, Event::Disp
     auto options = cluster->info()->extensionProtocolOptionsTyped<ProtocolOptionsConfigImpl>(
         NetworkFilterNames::get().RedisProxy);
     if (options) {
-      auth_password_ = options->auth_password(parent_.api_);
+      const_cast<std::string&>(auth_password_) = options->auth_password(parent_.api_);
```

Comment:
Sorry please revert this part of the change. If you can't set it inside an initializer list just leave it non-const.

Reply:
ok